dandi/publish/views/stats.py

```diff
@@ -0,0 +1,28 @@
+from django.contrib.auth import get_user_model
+from django.db.models import Sum
+from django.http import JsonResponse
+from rest_framework.viewsets import ViewSet
+
+from dandi.publish.models import Asset, DraftVersion, Version
+
+
+class StatsViewSet(ViewSet):
```

Comment:
I don't think this is a `ViewSet`, since it's not really exposing semantics of `list`, `retrieve`, `create`, etc.

It could be [an `APIView` with a `get`](https://www.django-rest-framework.org/api-guide/views/#class-based-views) if we anticipate supporting other HTTP verbs.

Otherwise, I think this should be [a function-based view](https://www.django-rest-framework.org/api-guide/views/#function-based-views).

Reply:
I was looking for something that did this and could not find it, thanks